### DISCUSSION TITLE
XA2 Pioneer upgrade message

### DISCUSSION THREAD

**Discussion by tneo** [Divested-Mobile/DivestOS-Build#281] [Q&A]
On the XA2 a message is shown that the current version can't be updated with a link to LineageOS to manually install. In the past this was not advised to do, but the divestos site reports the current build to be broken for XA2. Is the XA2 at the end of the support line?

**Comment by SkewedZeppelin** [comment]
I still have not found a fix for the boot issue after the August/September update.

**Comment by tneo** [comment]
I have a spare XA2, that I can play with. Is there something I can do to help you make a fix?
